tsconfig.json

```diff
@@ -34,6 +35,7 @@
   "include": [
     "src/**/*.ts",
     "src/**/*.vue",
-    "test/**/*.ts"
+    "test/**/*.ts",
+    "types/**/*.d.ts"
```

Comment:
What is this `**` notation

Reply:
glob, to recursively search subdirectories